sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -673,6 +675,27 @@ object DataSourceStrategy
     (nonconvertiblePredicates ++ unhandledPredicates, pushedFilters, handledFilters)
   }
 
+  protected[sql] def translateAggregate(
+      aggregates: AggregateExpression,
+      pushableColumn: PushableColumnBase): Option[AggregateFunc] = {
```

Comment:
We can always use `PushableColumnAndNestedColumn`, as this is a new API and we don't need a config to turn on nested column support: we can always support it.